web/server/entrypoint_test.go

```diff
@@ -158,15 +159,11 @@ func TestShutdown(t *testing.T) {
 		}
 		test.That(t, success, test.ShouldBeTrue)
 
-		conn, err := robottestutils.Connect(port)
-		test.That(t, err, test.ShouldBeNil)
-		defer func() {
-			test.That(t, conn.Close(), test.ShouldBeNil)
-		}()
-		rc := robotpb.NewRobotServiceClient(conn)
+		addr := "localhost:" + strconv.Itoa(port)
+		rc := robottestutils.NewRobotClient(t, testLogger, addr, time.Second)
 
 		testLogger.Info("Issuing shutdown.")
-		_, err = rc.Shutdown(context.Background(), &robotpb.ShutdownRequest{})
+		err = rc.Shutdown(context.Background())
```

Comment:
```suggestion
		err = rc.Shutdown(context.Background())
		test.That(t, err, test.ShouldBeNil)
```

Reply:
not guaranteed to be nil; err is checked with this line:
`test.That(t, isExpectedShutdownError(err, testLogger), test.ShouldBeTrue)`

Reply:
ah sorry, forgot about that